app/components/map/_MapComponent.client.tsx

```diff
@@ -119,6 +119,8 @@ export default function MapComponent() {
           setStartDate={setStartDate}
```

Comment:
Remember to remove `period` from here as well 👆🏻

Reply:
done! Hopefully I didn't miss it elsewhere.

Reply:
You kind of did, but that's fine.